include/celerity.h

```diff
@@ -24,4 +24,14 @@ namespace runtime {
 } // namespace runtime
 } // namespace celerity
 
+// Expose SYCL 2020 ::sycl namespace to Celerity users.
+// Implementations currently don't agree on whether the legacy include path <CL/sycl.hpp> should also
+// expose the new namespace. See https://github.com/KhronosGroup/SYCL-Docs/issues/137.
+// TODO: Remove this once Celerity includes <sycl/sycl.hpp> internally.
+#if WORKAROUND_HIPSYCL
+namespace sycl = cl::sycl;
```

Comment:
I think we should really `#include <SYCL/sycl.hpp>` here instead.
When declaring a namespace alias as it's done here, it is illegal to re-open the namespace with `namespace sycl { ... }` afterwards, as a user might want to do to specialize trait types, for example.

Reply:
You're right, that makes more sense! Fixed.